### PR TITLE
add doc page for browser package

### PR DESCRIPTION
**Pull request by digitalsadhu**

### CODE REVIEW

**Comment by trygve-lie** [comment]
Sweet. 

Could we though use a little bit different structure? I would like this to live on `https://podium-lib.io/docs/api/browser`.

Even if there now are just one feature in there, the Message Bus, I would like to make it clear that `@podium/browser` is a library for the browser and that the Message Bus is one component in this browser library.

**Comment by digitalsadhu** [comment]
I feel like there are maybe 2 user stories here:

* I have this problem (of cross browser comms) and how would I solve that in Podium
* I'm using the browser package, what are the methods and how do I use them

The first maybe should be a "guide" in the podlet section (what I have now) while the latter should be api docs under a browser section. How does that sound?

File: docs/podlet/podlet_to_podlet_communication.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+id: podlet_to_podlet_communication
+title: Podlet To Podlet Communication In The Browser
+---
+
+For situations where 2 or more podlets need to communicate with each other without a page refresh, Podium provides a client side pubsub library called `EventBus`.
```
Comment (alexanbj) [inline docs/podlet/podlet_to_podlet_communication.md]: We do not provide a library called `EventBus`

File: docs/podlet/podlet_to_podlet_communication.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+To illustrate by way of an example, if podlet A. contains an input field wherein a user can input a new reminder and podlet B. contains a list of all reminders, when a user inputs a new reminder into the area of the page that is rendered by podlet A. he or she would expect that the reminders list controlled by podlet B. would immediately be updated with the newly input reminder.
+
+`EventBus` provides a simple mechanism for a podlet to publish events or to subscribe to receive events, in the browser, whenever they occur. To continue with our example above, podlet B. would subscribe to receive new reminder events so that when a user inputs a new reminder, Podlet A. would publish a new reminder event. Podlet B. would then immediately receive that event and be able to react by updating the reminder list with a new reminder.
```
Comment (alexanbj) [inline docs/podlet/podlet_to_podlet_communication.md]: `EventBus` mentioned again

File: docs/podlet/podlet_to_podlet_communication.md (hunk 1 above)
Comment (digitalsadhu) [reply docs/podlet/podlet_to_podlet_communication.md]: true

Comment (digitalsadhu) [reply docs/podlet/podlet_to_podlet_communication.md]: @alexanbj 
You happy with this?:

Podium provides a client side library called `@podium/browser` which provides a pubsub class called `EventBus`.

Comment (alexanbj) [reply docs/podlet/podlet_to_podlet_communication.md]: It's better, but it there still no such thing as `EventBus` in that library ;)

Comment (digitalsadhu) [reply docs/podlet/podlet_to_podlet_communication.md]: oh geez, `MessageBus`.

**Pull request merged**